### PR TITLE
Capstone Project 2

### PR DESCRIPTION
**Pull request by eudondian**
This project was carried out by @Abass-rashid and @eudondian

The following expected functions are functioning in the app.

A home page shows a list of items you can "like."
A popup window with more data about an item that you can use to comment on it or reserve it for some time
TVmaze API: data about TV series and movies was used
When the popup loads, the web app retrieves data
The selected API shows details about the selected item.
The Involvement API shows the item comments.
The counter shows the number of items (home) and the number of comments (comments popup).
A descriptive README.md file was added to the project.
A video showing the description of the project is attached to the README.md file.

### CODE REVIEW

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have the right linter setup, kindly ensure you are using javascript HTML and CSS linter setup for your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+  liToRemove.forEach((item) => {
+    item.remove();
+  });
+  movies.forEach((movie, id) => {
+    const li = document.createElement("li");
+    li.classList.add("movie");
+    li.id = `li${movie.id}`;
+    li.innerHTML = `<img src="${movie.image.medium}" class="movie-img" alt="${movie.name}">`
+      + `<div class="movie-name-section"><p> ${movie.name}</p><i class="likeBtn fa fa-heart-o" id="${movie.id}" style="font-size:24px"></i></div>`
+      + `<div class="like-div"><p class="likes" id="like${movie.id}">${movie.likes}</p><p> Likes</p></div>`
```
Comment (Ismailco) [inline src/index.js:137]: - [ ] The movie likes button should add one like at a time instead it also increments "1" to the previous number, kindly fix this so the like button works as expected

Screenshot_20220804_173825

File: README.md [hunk 2]
```diff
@@ -1,62 +1,75 @@
...
-- Technologies used
+- HTML
+- CSS
+- JavaScript
+- Webpack
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link]([Live Demo Link](https://abass-rashid.github.io/capstone-project-2/))
```
Comment (Ismailco) [inline README.md]: - [ ] [OPTIONAL] The live demo doesn't open the website, instead, it displays the README file. please make sure the branch you using is hosted on Github pages, and target the right file or folder (/dist)

File: README.md [hunk 3]
```diff
@@ -1,62 +1,75 @@
...
 
+Clone the repo to your local machine
+run npm install to download the necessary modules.
+run npm start to load the project
+
 ### Install
 
 ### Usage
 
 ### Run tests
```
Comment (Ismailco) [inline README.md]: - [ ] [OPTIONAL] Kindly remove unused headers from your readme file, this will make your documentation more professional

**Review by Ismailco — CHANGES_REQUESTED** [review]
#### Hi @Abass-rashid, @eudondian,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by ShadyShawkat — APPROVED** [review]
## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**